automation/features/step-definitions/home.steps.js

```diff
@@ -50,10 +50,14 @@ Then(/^staked token field is showing as "([^"]*)?"$/, (amount) => {
   expect(stakedToken.getText()).toBe(amount);
 });
 
-Then(/^the vega wallet link is correct$/, async () => {
-  await expect(HomePage.vegaWalletLink).toHaveAttribute(
+Then(/^the vega wallet link is correct$/, () => {
+  HomePage.vegaWalletLink.waitForDisplayed({
```

Comment:
chore (blocking): HomePage here is an instance and not a class so it should be camelCase not PascalCase